stustapay/core/config.py

```diff
@@ -35,12 +35,20 @@ class CoreConfig(BaseModel):
     sumup_affiliate_key: str = "unset"
 
 
+class SEPAConfig(BaseModel):
+    sender_name: str
+    sender_iban: str
+    currency: str = "EUR"  # ISO 4217
```

Comment:
This is available as a config entry in the database, should not be configured in two places